Agents/Xamarin.Interactive/CodeAnalysis/Evaluating/EvaluationContextId.cs

```diff
@@ -19,7 +19,7 @@ internal static EvaluationContextId Create ()
             => this.id = id;
 
         public bool Equals (EvaluationContextId id)
-            => id.id == this.id;
+            => this.id.Equals (id.id);
```

Comment:
Reminder: this change is to work around a bug in WASM.

Reply:
Cool. When you rebase all this before merge, keep the changes to these to files as a separate commit so we can revert when we bump.